ui/mods/mod_dynamic_perks/hooks/screens/character/modules/character_screen_right_panel/character_screen_perks_module.js

```diff
@@ -55,6 +83,50 @@ CharacterScreenPerksModule.prototype.initPerkTree = function (_perkTree, _perksU
 	}
 };
 
+DynamicPerks.Hooks.CharacterScreenPerksModule_attachEventHandler = CharacterScreenPerksModule.prototype.attachEventHandler;
+CharacterScreenPerksModule.prototype.attachEventHandler = function(_perk)
+{
+	DynamicPerks.Hooks.CharacterScreenPerksModule_attachEventHandler.call(this, _perk);
+	var self = this;
+
+	_perk.Container.on('mouseenter.dynamicperks focus.dynamicperks' + CharacterScreenIdentifier.KeyEvent.PerksModuleNamespace, null, this, function (_event)
+	{
+		if (!MSU.getSettingValue("mod_dynamic_perks", "PerkTree_HighlightPerkGroups")) return;
+
+		$.each(_perk.PerkGroupIDs, function(_idx, _id){
+			$.each(self.mPerkGroups[_id], function(_, _innerPerk){
+				if (_perk ==_innerPerk) return;
+				_innerPerk.PerkGroupOverlay.css("border", "2px solid " + self.mPerkGroupColors[_idx]);
```

Comment:
I don't like how the size of the border is hidden in the .js code instead of being listed in the .css file.
Have you tried other approaches where you for example flip the `opacity` instead in the .js code or add a new .css class `perk-grou-highlighting active` which you then either add or remove from the PerkGroupOverlay?

In the .css
```
.perk-group-highlight.active {
    border: 2px solid rgba(255, 234, 125, 0.5);
}
```
This way the color would also move over to the .css which may be preferred while it's hard-coded anyways

In the .js
```
[...]
_innerPerk.PerkGroupOverlay.addClass("perk-group-highlight active");
[...]
_perk.PerkGroupOverlay.removeClass("perk-group-highlight active");
[...]
```

Reply:
The problem is that the color will need to be changed based on the index. Now we could add a bunch of color preset classes, like `.dynamicperks-image-overlay .color-preset-1`, `.dynamicperks-image-overlay .color-preset-2` ... But I don't think that's very nice either.